example_application.yml

```
@@ -0,0 +1,9 @@
+api_keys:
+  - token: "secret"
+
+weaviate:
+  host: "localhost"
+  port: "8001"
+  grpc-port: "50051"
```

Otherwise pydantic cannot resolve this properly
```suggestion
  grpc_port: "50051"
```